.github/workflows/rust.yml

```diff
@@ -79,3 +79,5 @@ jobs:
         with:
           command: test
           args: --test loom --features=loom,instrumentation --release --verbose
+          env:
+            LOOM_MAX_PREEMPTIONS: 3
```

Comment:
We limit here the number of context switch to keep the number of sequence of states that Loom checks to an amount compatible with a usable CI runtime.